### PR TITLE
Consuming backend API from frontend

### PR DESCRIPTION
**Pull request by Michaeljogoh**
Consuming backend api for frontend , backend and frontend is now connect with axios package. User can  make a post method and data is store in database.

### CODE REVIEW

File: text.http [hunk 1]
```diff
@@ -1,4 +1,7 @@
-GET  http://localhost:5000/users/post
+POST  http://localhost:5000/users/comment
```
Comment (iamr0b0tx) [inline text.http:1]: why users?

File: frontend/src/component/Post.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+const Post = () => {
+
+const [title , setTitle] = useState('');
+const [body , setBody] = useState('');
+const [author, setAuthor] = useState('');
+//Intergrating the backend API
+async function postBlog(e){
+  e.preventDefault()
+  try {
+    await axios.post("http://localhost:5000/users/post", {
```
Comment (iamr0b0tx) [inline frontend/src/component/Post.js:14]: don't hardcode values like this. make them into constants in a separate file

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: text.http (hunk 1 above)
Comment (Michaeljogoh) [reply text.http:1]: The users was used  because of the router.get

File: frontend/src/component/Post.js (hunk 2 above)
Comment (Michaeljogoh) [reply frontend/src/component/Post.js:14]: Ok sir , I'll make sure I do that tomorrow.

**Pull request merged**